pkg/operator/staticpod/controller/installer/installer_controller_test.go

```diff
@@ -317,7 +317,7 @@ func TestNewNodeStateForInstallInProgress(t *testing.T) {
 			&operatorv1.NodeStatus{
 				NodeName:        "test-node-1",
 				CurrentRevision: 0,
-				TargetRevision:  0,
+				TargetRevision:  2,
```

Comment:
this comes from https://github.com/openshift/library-go/pull/1151/commits/db266aea60807d3b8678f469dfbcc2a04a988d2f#r675510545 because now the node state func sets target revision when there is a revision pending. Before the sync func with the fallthrough did this and we stayed at zero target version here in the tests.

Same for all 4 cases here and below.